app/routes/wow.weekly-price-group-delta.tsx

```
@@ -0,0 +1,1256 @@
+            <div className="bg-white dark:bg-gray-800 p-4 rounded-lg shadow mb-4">
+              <button
+                onClick={() =>
+                  setShowPriceQuantityCharts(!showPriceQuantityCharts)
+                }
+                className="bg-blue-500 hover:bg-blue-600 text-white px-4 py-2 rounded-lg shadow-md transform transition-all duration-200 hover:scale-105 flex items-center gap-2">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  className="h-5 w-5"
+                  viewBox="0 0 20 20"
+                  fill="currentColor">
+                  <path
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)